Makefile

```diff
@@ -7,20 +7,19 @@ export GO15VENDOREXPERIMENT=1
 VERSION := 0.0.1-$(shell date "+%Y%m%d%H%M%S")
 LDFLAGS := "-s -X main.version=${VERSION}"
 BINDIR := ./rootfs/bin
-DEV_REGISTRY ?= $(eval docker-machine ip deis):5000
+DEV_REGISTRY ?= $(docker-machine ip deis):5000
```

Comment:
What about on Linux systems that may have docker running natively and don't have docker-machine installed?

See the following for discussion and possible fix:

https://github.com/deis/prototype-repo/issues/8
https://github.com/deis/prototype-repo/pull/11